contrib/libhdfs3-cmake/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-if(NOT ARCH_AARCH64 AND NOT OS_FREEBSD AND NOT APPLE AND NOT ARCH_PPC64LE AND NOT ARCH_S390X)
+if(NOT OS_FREEBSD AND NOT APPLE AND NOT ARCH_PPC64LE AND NOT ARCH_S390X)
```

Comment:
What about supporting HDFS on Mac? Probably it can be just enabled and it will work?

Reply:
Doable but a bit more complicated (especially curl on Mac is tricky). [Another PR](https://github.com/ClickHouse/ClickHouse/pull/51276) attempts this.

Reply:
Yeah, missed it, sorry